geometry_msgs/Pose2D tracking_pose
geometry_msgs/Pose2D[] future_poses
float64 tracking_speed
bool reset_sim
bool path_planner_terminate
bool end_of_action
float64 action_progress
bool auto_pilot
int64 scenario_chosen
